### PR TITLE
Initialize the DB with CDK

### PR DESCRIPTION
**Pull request by beekley**
## Description

Addresses: [Set up the map data DB](https://app.shortcut.com/blueconduit/story/3718/set-up-the-map-data-db)

This sets up basic networking and data plane stacks for the app.

### New

* The data plane stack consists of a serverless Aurora instance running Postgres.
* The network stack consists of a VPC.
* The root stack builds the above stacks.

### Changed

* File structure.

### Removed

* n/a

## Testing and Reviewing

Run `cdk deploy --all` to test. Actual unit tests to come in a future PR.

### CODE REVIEW

File: cdk/src/cdk.ts [hunk 1]
```diff
@@ -13,9 +15,9 @@ new OpenDataPlatformStack(app, 'CdkStack', {
    * and Region that are implied by the current CLI configuration. */
   // env: { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION },
 
-  /* Uncomment the next line if you know exactly what Account and Region you
-   * want to deploy the stack to. */
-  // env: { account: '123456789012', region: 'us-east-1' },
+  // Development account.
+  // TODO: replace with AWSGoogleSandbox account.
```
Comment (kailamjeter) [inline cdk/src/cdk.ts:19]: Is this just for default config or do we generally want to move to deploying to AWSGoogleSandbox? My understanding was that we would be deploying the actual project to BlueConduit's accounts and that the sandbox account was just for us to experiment with cdk tutorials.

**Review by kailamjeter — APPROVED** [review]

**Review by justinebreuch — APPROVED** [review]
Looks good, but curious to hear more about how you went about this!

File: cdk/src/open-data-platform/data-plane-stack.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const serverlessCluster = new rds.CfnDBCluster(this, 'serverlessCluster', {
+      dbClusterIdentifier: `main-aurora-serverless-cluster`,
+      engineMode: 'serverless',
+      engine: 'aurora-postgresql',
+      engineVersion: '10.18',
+      enableHttpEndpoint: true,
+      databaseName: 'main',
+      dbSubnetGroupName: clusterSubnets.ref,
+      masterUsername: 'adminuser',
+      masterUserPassword: 'blueconduit', // TODO: GENERATE THIS!
```
[the remark this answers is not in the document]
Comment (beekley) [reply cdk/src/open-data-platform/data-plane-stack.ts]: Done. The readme describes how to use the new generated credentials.

File: cdk/src/cdk.ts (hunk 1 above)
Comment (beekley) [reply cdk/src/cdk.ts:19]: That's a good question that I don't know the answer to. I'm going to leave this comment in there temporarily until we figure that out, but I want to submit this in the meantime.

**Pull request merged**